.circleci/main.yml

```diff
@@ -217,12 +217,18 @@ jobs:
       - *store_gomod
   interop:
     docker:
-      - image: cimg/go:1.16-node
```

Comment:
Are you sure we don't need Go? IIRC we needed it in order to build custom Go binaries like the relay daemon or go-ipfs.

Reply:
Affirmative: binary (`IPFS_GO_EXEC: /tmp/circleci-workspace/bin/ipfs`) is built in `build` step:
https://github.com/ipfs/go-ipfs/blob/467fb0af8c460af76b2c54fe0f871576f1252ea7/.circleci/main.yml#L382-L384

Reply:
But to make it easier to work with custom binaries, restored use of `cimg/go:1.16-node`